src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -202,4 +202,22 @@ public function count()
     {
         return \count($this->parameters);
     }
+
+    /**
+     * Returns the parameter value converted to array.
+     *
+     * @param string $key
+     * @param array $default
+     * @return array
+     */
+    public function getJsonArray($key, $default = [])
+    {
+        $value = $this->get($key, $default);
+
+        if (is_string($value)) {
+            $value = json_decode ($value);
```

Comment:
1) If you pass true as second parameter, you wouldn't need cast `$value` to array at line 221 .
2) redundant space after `json_decode`